share_extension/android/extension_post/extension_post.js

```diff
@@ -406,17 +406,30 @@ export default class ExtensionPost extends PureComponent {
     onPost = () => {
         const {channelId, files, value} = this.state;
         const {currentUserId} = this.props;
+        const {formatMessage} = this.context.intl;
 
-        const data = {
-            channelId,
-            currentUserId,
-            files,
-            token: this.token,
-            url: this.url,
-            value,
-        };
-
-        this.onClose(data);
+        if (value.length > MAX_MESSAGE_LENGTH) {
+            Alert.alert(
+                formatMessage({
+                    id: 'mobile.message_length.message',
+                    defaultMessage: 'Your current message is too long. Current character count: {max}/{count}',
```

Comment:
should be count / max

Reply:
I agree... I thought it was odd too, but went with the pre-existing convention. Good to know that it was dealt with recently. I've just now matched this PR to follow what the corrected format. Thanks @enahum for saving me the time to go hunting for what changed between when I opened this PR to now 👍